File: tests/dir_cases/test1-cpp-expected/foo.cpp
#include <cassert>  // NOLINT(build/include_order)
#include <cstdint>  // NOLINT(build/include_order)

#include "bar.h"
#include "baz.h"
int main(int argc, char** argv) {
  int x = bar1();
  std::string y = baz1();
  assert(x == 0);
  assert(y == std::string{"foo"});
}

File: tests/dir_cases/test1-go-expected/foo.go
package main

import (
	"bar/bar1"
)
import (
	"baz/baz1"
)

func main() {
	x := bar1()
	y := baz1()
	if !(x == 0) {
		panic("assert")
	}
	if !(y == "foo") {
		panic("assert")
	}
}
